src/components/application_manager/src/state_controller_impl.cc

```diff
@@ -1270,15 +1270,7 @@ mobile_apis::VideoStreamingState::eType StateControllerImpl::CalcVideoState(
     const mobile_apis::HMILevel::eType hmi_level) const {
   auto state = mobile_apis::VideoStreamingState::NOT_STREAMABLE;
 
-  if (app->webengine_projection_enabled()) {
```

Comment:
@AByzhynar why did you remove this? Wrong behavior found by a new unit tests?

Reply:
@AKalinich-Luxoft It was redundant at it is covered by IsVideoApplication method